apps/admin/backend/src/app.ts

```diff
@@ -292,25 +297,30 @@ function buildApi({
 
     async addCastVoteRecordFile(input: {
       path: string;
-    }): Promise<AddCastVoteRecordFileResult> {
+    }): Promise<
+      Result<
+        Admin.CvrFileImportInfo,
+        AddCastVoteRecordReportError & { message: string }
+      >
+    > {
       const { path } = input;
       const userRole = assertDefined(await getUserRole());
       const filename = basename(path);
       let fileStat: Stats;
       try {
         fileStat = await fs.stat(path);
       } catch (error) {
+        const message = `Failed to access cast vote record report for import.`;
```

Comment:
I copied it just because it to generate the message with `getAddCastVoteRecordReportErrorMessage` I would have had to sort of manufacture an error to pass into the method. But I can do that instead. Changed in e191dddc28eefaf40b3e17a77615d65c8c74a297